vllm/v1/worker/tpu_model_runner.py

```diff
@@ -846,3 +868,11 @@ def _get_padded_token_len(x: int) -> int:
     if x <= 16:
         return 16
     return 1 << (x - 1).bit_length()
+
+
+def _get_padded_num_reqs(max_num_reqs, x: int = 1) -> int:
+    if x > max_num_reqs:
+        return max_num_reqs
+    if x <= 64:
+        return 64
+    return 1 << (x - 1).bit_length()
```

Comment:
Is our minimum num_reqs now 64? This seems rather large for a minimum

Reply:
No, change the logic a bit and now the upper limit is `max_num_reqs`
```
res = 64 if x <= 64 else 1 << (x - 1).bit_length()
return min(res, upper_limit)
```